### DISCUSSION TITLE
/r is added to a replaceString resulting in wrong result

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#32608] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

gitlab

### Please tell us more about your question or problem

Renovate configuration:
```json
      "matchStrings": [
        "source-image: \"(?<depName>.*):(?<currentValue>.*)\"",
        "bootc switch --mutate-in-place --transport registry (?<depName>.*):(?<currentValue>.*)\\n"
      ],
```

The file content is:
```
[customizations.installer.kickstart]
contents = """
services --disabled=cloud-init

reboot --eject
%post
bootc switch --mutate-in-place --transport registry registry.redhat.io/rhelai1/bootc-nvidia-rhel9:1.3
touch /etc/cloud/cloud-init.disabled
%end
"""
```

Renovate doesn't identify this as a dependency

### Logs (if relevant)

<details><summary>Logs</summary>

```

**Comment by lmilbaum** [comment]
A snapshot of the log file. When copy pasting the `\r` disappears.

<img width="1175" alt="Screenshot 2024-11-19 at 1 13 56" src="https://github.com/user-attachments/assets/eb423a80-6ec6-4e44-a27c-8737f7c8c556">

**Comment by rarkins** [comment]
Are you running on windows?

**Reply by lmilbaum** [reply]
Nop

**Reply by lmilbaum** [reply]
Just noticed that I have a `^M` at the end of the sentence

**Reply by rarkins** [reply, marked as answer]
\r\n usually comes from windows machines. Potentially someone using windows managed to commit it to the repo then
